### ISSUE TITLE
Accessibility: Invalid BoundingRectangle value for CheckedListBox entries

### ISSUE THREAD

**Issue by vladimir-krestov** [dotnet/winforms#1135]
Steps to reproduce:
1. Create a new .Net Core WinForms app
2. Add to app the CheckedListBox control
3. Add to CheckedListBox some items
4. Open the Inspect tool and check the BoundingRectangle of CheckedListBox

Actual result:
The Inspect shows the bounds of invisible items.
![CheckedListBoxBounds](https://user-images.githubusercontent.com/49272759/59595862-4dedc880-90ff-11e9-8bbe-07ba584b70d8.png)

Expected result:
BoundingRectangle for all entries should be within a parent element.
